### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bradparks**
This simply adds how to install it to the readme, which is not necessarily obvious to people that don't use node a ton!

-----
[View rendered README.md](https://github.com/bradparks/rest.js/blob/patch-1/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,9 @@
 [![Coverage Status](https://coveralls.io/repos/github/octokit/rest.js/badge.svg)](https://coveralls.io/github/octokit/rest.js)
 [![Greenkeeper](https://badges.greenkeeper.io/octokit/rest.js.svg)](https://greenkeeper.io/)
 
+## Installation
+```$ npm install --save @octokit/rest```
```
Comment (gr2m) [inline README.md]: Could you please make it a code block? You’ll have to put the \`\`\` into their own lines.

Remove the `$`, it makes it easier to copy without it. Remove the `--save`, it’s no longer needed.

**Review by gr2m — CHANGES_REQUESTED** [review]
Thanks for the pull request! I removed installation information recently when I updated the website: https://octokit.github.io/rest.js/. But we can leave it in the README so people can see it on first glance.

Just have a few change requests. Thanks!

File: README.md (hunk 1 above)
Comment (wolfy1339) [reply README.md]: It's better to be explicit here, as some people disable that functionality. Plus it's needed for older versions of npm

Comment (gr2m) [reply README.md]: It’s default behaviour for all supported node/npm versions. People who disable that functionality know what they are doing. Also `--save` is now `--save-prod` as far as I know. I would leave it out.

Comment (bradparks) [reply README.md]: I think I made the changes....

**Review by gr2m — APPROVED** [review]
Thanks Brad!

**Pull request merged**